akka-docs/src/main/paradox/typed/actor-lifecycle.md

```diff
@@ -16,7 +16,40 @@ TODO intro
 
 ## Creating Actors
 
-TODO
+Actor hierarchy consists of two kinds of actors – a root actor and its children. While there is only one root actor per
+@unidoc[akka.actor.typed.ActorSystem], the root actor can create, or _spawn_, arbitrary number of child actors which can
+in turn spawn child actors of their own.
+
+The root actor, also called the guardian actor, is created along with the `ActorSystem`. Messages sent to the actor system are directed to the root actor.
+The root actor is defined by the behavior used to create the `ActorSystem`, named `HelloWorldMain.main` in the example below:
+
+Scala
+:  @@snip [IntroSpec.scala]($akka$/akka-actor-typed-tests/src/test/scala/docs/akka/typed/IntroSpec.scala) { #hello-world }
+
+Java
+:  @@snip [IntroSpec.scala]($akka$/akka-actor-typed-tests/src/test/java/jdocs/akka/typed/IntroTest.java) { #hello-world }
+
+
+Child actors are spawned with @unidoc[akka.actor.typed.ActorContext]'s `spawn`. In the example below, when the root actor
+is started, it spawns a child actor described by the behavior `HelloWorld.greeter`. Additionally, when the root actor receives a
+`Start` message, it creates a child actor defined by the behavior `HelloWorldBot.bot`:
+
+Scala
+:  @@snip [IntroSpec.scala]($akka$/akka-actor-typed-tests/src/test/scala/docs/akka/typed/IntroSpec.scala) { #hello-world-main }
+
+Java
+:  @@snip [IntroSpec.scala]($akka$/akka-actor-typed-tests/src/test/java/jdocs/akka/typed/IntroTest.java) { #hello-world-main }
+
+By default, child actors will use the same dispatcher as their parent actor. Actor dispatcher can be customized
+through @unidoc[akka.actor.typed.Props] or @unidoc[DispatcherSelector]:
```

Comment:
I want this to be true, but I am not sure it is, can you please verify?

Reply:
you are right - it is not true, the dispatcher defaults to `akka.actor.default-dispatcher`, not to parent's. Will fix shortly. I got misguided by the `DispatcherSelector` scaladoc

Reply:
I think it was true for the native typed actor system which we have put on ice for now.

Reply:
If you could update the DispatcherSelector scaladoc as well that'd be great.

Reply:
sure, fix update